src/libs/actions/Report.js

```diff
@@ -969,7 +807,7 @@ function handleReportChanged(report) {
     // A report can be missing a name if a comment is received via pusher event
     // and the report does not yet exist in Onyx (eg. a new DM created with the logged in person)
     if (report.reportID && report.reportName === undefined) {
-        fetchChatReportsByIDs([report.reportID]);
+        openReport(report.reportID);
```

Comment:
This is the only risky change – need to make extra sure that `openReport` will do everything that `fetchChatReportsByIDs` once did